vue.config.js

```diff
@@ -27,10 +31,10 @@ module.exports = {
       poll: true
     },
     host: '0.0.0.0',
-    port: 8000,
```

Comment:
Not sure why I didn't just set this to the value...

```suggestion
    port: vueTorrentPort,
```

Whoops :sweat_smile: